tutorial_code/example0_1_orbits.py

```diff
@@ -43,49 +43,55 @@ def make_bqm():
+
     bqm = make_bqm()
     signed_bqm = orbits.make_signed_bqm(bqm)
     signed_qubit_orbits, signed_coupler_orbits = orbits.get_bqm_orbits(signed_bqm)
     qubit_orbits, coupler_orbits, qubit_orbits_opposite, coupler_orbits_opposite = \
         orbits.get_orbits(bqm)
+    if verbose:
+        print('Signed qubit orbits: {signed_qubit_orbits}')
+        print(f'\nSigned coupler orbits: {signed_coupler_orbits}')
+        print(f'\nQubit orbits: {qubit_orbits}')
+        print(f'\nCoupler orbits: {coupler_orbits}')
 
-    print(f'Signed qubit orbits: {signed_qubit_orbits}')
-    print(f'\nSigned coupler orbits: {signed_coupler_orbits}')
-    print(f'\nQubit orbits: {qubit_orbits}')
-    print(f'\nCoupler orbits: {coupler_orbits}')
+        print('\nQubit orbits opposite:')
+        for p, q in enumerate(qubit_orbits_opposite):
+            print(f'QubitOrbit{p} = -QubitOrbit{q}')
 
-    print('\nQubit orbits opposite:')
-    for p, q in enumerate(qubit_orbits_opposite):
-        print(f'QubitOrbit{p} = -QubitOrbit{q}')
-
-    print('\nCoupler orbits opposite:')
-    for p, q in enumerate(coupler_orbits_opposite):
-        print(f'CouplerOrbit{p} = -CouplerOrbit{q}')
+        print('\nCoupler orbits opposite:')
+        for p, q in enumerate(coupler_orbits_opposite):
+            print(f'CouplerOrbit{p} = -CouplerOrbit{q}')
 
     qubit_orbit_sizes = [len([x for x in qubit_orbits.values() if int(x) == i]) for i in
                          range(len(qubit_orbits_opposite))]
     coupler_orbit_sizes = [len([x for x in coupler_orbits.values() if int(x) == i]) for i in
                            range(len(coupler_orbits_opposite))]
-    
-    print(f'Qubit orbit sizes: {qubit_orbit_sizes}')
-    print(f'Coupler orbit sizes: {coupler_orbit_sizes}')
+    if verbose:
+        print(f'Qubit orbit sizes: {qubit_orbit_sizes}')
+        print(f'Coupler orbit sizes: {coupler_orbit_sizes}')
 
-    for i in range(len(qubit_orbit_sizes)):
-        if qubit_orbit_sizes[i] > 0 and qubit_orbit_sizes[qubit_orbits_opposite[i]] > 0:
-            print(f'Qubit orbit {i} has behavior opposite to qubit orbit {qubit_orbits_opposite[i]}.')
+        for i in range(len(qubit_orbit_sizes)):
+            if qubit_orbit_sizes[i] > 0 and qubit_orbit_sizes[qubit_orbits_opposite[i]] > 0:
+                print(f'Qubit orbit {i} has behavior opposite to qubit orbit {qubit_orbits_opposite[i]}.')
 
-    for i in range(len(coupler_orbit_sizes)):
-        if coupler_orbit_sizes[i] > 0 and coupler_orbit_sizes[coupler_orbits_opposite[i]] > 0:
-            print(f'Coupler orbit {i} behavior opposite to coupler orbit {coupler_orbits_opposite[i]}.')
+        for i in range(len(coupler_orbit_sizes)):
+            if coupler_orbit_sizes[i] > 0 and coupler_orbit_sizes[coupler_orbits_opposite[i]] > 0:
+                print(f'Coupler orbit {i} behavior opposite to coupler orbit {coupler_orbits_opposite[i]}.')
 
     plt.rc('font', size=12)
```

Comment:
We could onsider modularize this part as a separate visualization function for code clarity.